Src/PCompiler/CompilerCore/Parser/PLexer.g4

```diff
@@ -82,6 +80,10 @@ RENAME         : 'rename' ;
 SAFE           : 'safe' ;
 MAIN		   : 'main' ;
 
+// machine queue annotations
```

Comment:
@lmpick can we drop these changes to the parser? are we using it in the PSymbolic engine?